images/installer/Dockerfile

```diff
@@ -4,8 +4,10 @@ MAINTAINER OpenShift Team <dev@lists.openshift.redhat.com>
 
 USER root
 
+ADD images/installer/origin.repo /etc/yum.repos.d/
```

Comment:
@jcantrill Actually could you just add the repo definition under images/installer/root/etc/yum.repos.d and move this section to the top:

```
# Add image scripts and files for running as a system container
COPY images/installer/root /
```